pipeline/api/cloud.py

```diff
@@ -332,7 +335,41 @@ def _post(self, endpoint, json_data):
 
         return response.json()
 
-    def _post_file(self, endpoint: str, file: io.BytesIO) -> FileGet:
+    def _patch(self, endpoint: str, json_data: dict) -> dict:
+        self.raise_for_invalid_token()
+        headers = {
+            "Authorization": "Bearer %s" % self.token,
+            "Content-type": "application/json",
+        }
+
+        url = urllib.parse.urljoin(self.url, endpoint)
+        response = httpx.patch(
+            url, headers=headers, json=json_data, timeout=self.timeout
+        )
+
+        if response.status_code == HTTPStatus.UNPROCESSABLE_ENTITY:
+            schema = json_data
+            raise InvalidSchema(schema=schema)
+        else:
+            self._get_raise_for_status(response)
+
+        return response.json()
+
+    def _delete(self, endpoint: str) -> dict:
+        self.raise_for_invalid_token()
+        headers = {
+            "Authorization": "Bearer %s" % self.token,
+            "Content-type": "application/json",
+        }
+
+        url = urllib.parse.urljoin(self.url, endpoint)
+        response = httpx.delete(url, headers=headers, timeout=self.timeout)
+        if response.status_code is not HTTPStatus.NO_CONTENT:
```

Comment:
`HTTPStatus.NO_CONTENT` is an enum whereas `response.status_code` is an integer, so comparison by `is` will fail.

```suggestion
        if response.status_code != HTTPStatus.NO_CONTENT:
```